Banner index

Javier359 opened the pull request:
El banner del proyecto con sus respectivas imagenes responsive

Commit:
se creo la rama Banner de la pagina index y se implemento la estructura del carrusel

Commit:
Se agregaron las imagenes de la carpeta banner al proyecto

Commit:
se quietaron los indicadores de la parte de abajo del carrusel

Commit:
se agregaron las clases img-fluid, se agregaron estilos a las imagenes en css

Commit:
Se agregaron mediasQuery para las imagens en dispositivos pequeños de 767px

brunomry reviewed (changes requested):
eliminar la palabra git del atributo class de las imagenes del carrusel
cambiar nombre de archivo style.css por index.css 
modificar id del carrusel y también en el archivo css
cambiar también nombre de archivo css en etiqueta link de html

Commit:
elimine la palabra git del atributo class de las imagenes del carrusel,
cambie nombre de archivo style.css por index.css,
modifique id del carrusel y también en el archivo css,
cambie también nombre de archivo css en etiqueta link de htm

brunomry reviewed (approved):
correcciones realizadas! 💯
